File: Balloon Everywhere.cpp
class Solution{
public:
    int maxInstance(string s){
        vector<int> v(26,0);
        for(int i=0;i<s.length();i++){
            v[s[i]-'a']++;
        }
        int ans=min({v['b'-'a'],v['a'-'a'],v['l'-'a']/2,v['o'-'a']/2,v['n'-'a']});
        return ans;
    }
};

File: Balloon Everywhere.py
class Solution:
    def maxInstance(self, s : str) -> bool:
        # code here
        diction = {"b":0,"a":0,"l":0,"o":0,"n":0}
        for i in s:
            if i in diction:
                diction[i]+=1
        diction["l"]//=2
        diction["o"]//=2
        return min(diction.values())
        
